source/glossary.rst

```diff
@@ -8,6 +8,12 @@ Glossary of Terms
      Headphone cueing, or just cueing, is listening to the next track you would
      like to mix in your headphones. The audience will not hear what you are
      cueing in your headphones. Being able to cue is a crucial aspect to DJing.
+     The shortcut for headphone cueing in Deck1 is T while for Deck 2 the shortcut is Y.
+
+   shortcuts
+     Shortcuts let you work more efficiently. This manual shows some shortcuts for 
+     accessing Mixxx functions with an English keyboard layout. Depending on your language
+     settings the defaults may be different.
```

Comment:
@ronso0 Are the keyboard shortcuts in the translated versions of the manual modified appropriately? If so, I don't know if this is needed.

Reply:
> Are the keyboard shortcuts in the translated versions of the manual modified appropriately?

I doubt that. Just checked the czech manual (which is hardly translated anyway) and there is no re-mapping of keys.
Also we shouldn't rely on translators updating the keys, so I think the hint here is okay.

Reply:
Done! and i used the singular form ```shortcut``` it fits best